sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -133,118 +59,32 @@ The authentication events trigger allows a function to be executed when an authe
-    "authenticationEventListenerId1": "10000000-0000-0000-0000-000000000001",
-    "customAuthenticationExtensionId": "10000000-0000-0000-0000-000000000002",
-    "authenticationContext1":{
-      "correlationId": "20000000-0000-0000-0000-000000000002",
-      "client": {
-        "ip": "127.0.0.1",
-        "locale": "en-us",
-        "market": "en-au"
-      },
-      "authenticationProtocol": "OAUTH2.0",
-      "clientServicePrincipal": {
-        "id": "40000000-0000-0000-0000-000000000001",
-        "appId": "40000000-0000-0000-0000-000000000002",
-        "appDisplayName": "Test client app",
-        "displayName": "Test client application"
-      },
-      "resourceServicePrincipal": {
-        "id": "40000000-0000-0000-0000-000000000003",
-        "appId": "40000000-0000-0000-0000-000000000004",
-        "appDisplayName": "Test resource app",
-        "displayName": "Test resource application"
-      },
-      "user": {
-        "companyName": "Nick Gomez",
-        "country": "USA",
-        "createdDateTime": "0001-01-01T00:00:00Z",
-        "displayName": "Dummy display name",
-        "givenName": "Example",
-        "id": "60000000-0000-0000-0000-000000000006",
-        "mail": "test@example.com",
-        "onPremisesSamAccountName": "testadmin",
-        "onPremisesSecurityIdentifier": "DummySID",
-        "onPremisesUserPrincipalName": "Dummy Name",
-        "preferredDataLocation": "DummyDataLocation",
-        "preferredLanguage": "DummyLanguage",
-        "surname": "Test",
-        "userPrincipalName": "testadmin@example.com",
-        "userType": "UserTypeCloudManaged"
-      }
-    }
-  }
-}
-```
+To test token augmentation, please do the following.
 
-* You should see this response:
-
-```json
-{
-    "data": {
-        "@odata.type": "microsoft.graph.onTokenIssuanceStartResponseData",
-        "actions": [
-            {
-                "@odata.type": "microsoft.graph.provideClaimsForToken",
-                "claims": {
-                    "DateOfBirth": "01/01/2000",
-                    "CustomRoles": [
-                            "Writer",
-                            "Editor"
-                        ]
-                    }
-             }
-        ]
-    }
-}
-```
+* Open the project that was created in the prior step. [How to get started](#how-to-get-started)
+* Follow [these steps](https://learn.microsoft.com/entra/identity-platform/custom-extension-tokenissuancestart-setup?tabs=visual-studio%2Cazure-portal&pivots=nuget-library#build-and-run-the-project-locally) to test your app locally.
 
 ## Troubleshooting
 
-* Visual Studio Code
-  * If running in Visual Studio Code, you get an error along the lines of the local Azure Storage Emulator is unavailable, you can start the emulator manually.! (Note: Azure Storage emulator is now deprecated and the suggested replacement is [Azurite](https://learn.microsoft.com/azure/storage/common/storage-use-azurite?tabs=visual-studio))
+### Visual studio code
```

Comment:
```suggestion
### Visual Studio Code
```

nit:  This is a product, and we need to reflect branding

Reply:
Updated